ystia/yorc
# Using session.CombinedOutput() instead of session.Run()

# Pull Request description

## Description of the change

Fixing an issue where the orchestrator got an empty output from a ssh command, while the output is not empty.
Should be related to a concurrency issue on a buffer used for both stdout and stderr in go func session.Run().

### What I did

Replacing the call session.Run() by a call to session.CombinedOutput() taking care of managing the write exclusive access to the buffer used for both stdout an stderr.

### How to verify it

Install Alien4Cloud 2.0.0 and a Yorc Orchestrator from this branch.
From Alien4Cloud, configure a Slurm location with an on-demand resource yorc.nodes.slurm.Compute.
Create a simple application with just a Compute node.
Deploy/Undeploy this application 10 times, and check each deployment/undeployment was successful.
